### ISSUE TITLE
Not An Issue 

### ISSUE THREAD

**Issue by ghost** [MrCheeze/playcoin#7]
What's Your Discord? I Wanna Send You Something Related To This App.

**Comment by MrCheeze** [comment]
is it the virus at the top of your github by any chance

**Comment by ghost** [comment]
> is it the virus at the top of your github by any chance

No. But Nice Joke. I Just Wanna Tell You Something In Your Discord DM's. Btw I Also Mentioned It's About Your Playcoin App.

**Comment by ghost** [comment]
> is it the virus at the top of your github by any chance

I Also Sent Friend Request To You. J.B. 3 With The Salad Is Me Btw
